src/components/Tables/ProductsTable/ProductsTable.js

```diff
@@ -47,13 +47,26 @@ const ProductsTable = () => {
 
   let result = products;
   if (searchTerm) {
-    result = products.filter((product) =>
-      product.description.toLowerCase().includes(searchTerm.toLowerCase())
-    );
+    result = products.filter((product) => {
+      var digits = product._id.toString();
+      if (digits.includes(searchTerm)) return product;
+      else if (
+        product.description.toLowerCase().includes(searchTerm.toLowerCase())
+      )
+        return true;
+      else if (product.category_name !== undefined) {
+        if (
+          product.category_name.toLowerCase().includes(searchTerm.toLowerCase())
+        )
+          return true;
+      } else if (product.stock_qty === searchTerm) return true;
+      else if (product.cost_with_tax === searchTerm) return true;
+      else if (product.unit_sell_price === searchTerm) return true;
```

Comment:
Sorry leels! I feel like you can take out the prices, but add the search for UPC and supplier code